tools/CACertificates/certGen.sh

```diff
@@ -0,0 +1,407 @@
+## Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is a total nit, but is it possible to name this script in kabob-case like the powershell script?

Reply:
Several pieces of our MS documentation use this script file after syncing this out from the C SDK repo. Believe me this naming convention does not work for me as well :) but I am not sure I can change the name.

Reply:
Ah. Gotcha. Ok, we should keep it.